docs/labs.md

```diff
@@ -66,8 +66,11 @@ An implementation of [MSC2241](https://github.com/matrix-org/matrix-doc/pull/224
 
 This also includes a new implementation of the user & member info panel, designed to share more code between showing community members & room members. Built on top of this new panel is also a new UX for verification from the member panel.
 
-## Cross-signing (`feature_cross_signing`)
+## Cross-signing (in development) (`feature_cross_signing`)
 
 Cross-signing ([MSC1756](https://github.com/matrix-org/matrix-doc/pull/1756))
 improves the device verification experience by allowing you to verify a user
 instead of verifying each of their devices.
+
+This feature is still in development and will be landing in several chunks in
+the coming weeks.
```

Comment:
putting a time on a static doc feels weird. Can we leave it with a full stop after "chunks"?

Reply:
Sure, works for me. It's hard to decide what's enough words for "not yet done" without being mysterious...

Reply:
Hopefully we're quite clear that if people have more questions they can ask in #riot-web